src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
-* Set your default Kubernetes context (this is required to use helm):
-** In MiniKube this is set for you automatically
-** Otherwise, you may have to modify the KUBECONFIG environment variable and then type `kubectl config use-context <my-context>`
+This section walks you through multi-user deployment of Che on Kubernetes.
+
+*Prerequisites*
+
+* A Kubernetes cluster with at least 4GB of RAM and RBAC:
+** For Minikube 0.26.0 and higher, use the `minikube start --cpus 2 --memory 4096 --extra-config=apiserver.authorization-mode=RBAC` command.
+** For Minikube 0.25.2 and lower, use the `minikube start --cpus 2 --memory 4096 --extra-config=apiserver.Authorization.Mode=RBAC` command.
+* To deploy role binding, use the `kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default` command.
+* Install the https://github.com/kubernetes/helm/blob/master/docs/install.md[Helm] CLI.
+* Set your default Kubernetes context (this is required to use Helm). In Minikube, this is set automatically. Otherwise, modify the `KUBECONFIG` environment variable and then type `kubectl config use-context <my-context>`.
 * Install tiller on your cluster:
-** Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount]: `kubectl create serviceaccount tiller --namespace kube-system`
-** Bind it to the almighty cluster-admin role: `kubectl apply -f ./tiller-rbac.yaml`
-** Install tiller itself: `helm init --service-account tiller`
-* Ensure that you have an NGINX-based ingress controller. Note: This is the default ingress controller on MiniKube. You can start it with `minikube addons enable ingress`
-* DNS discovery should be enabled. Note: It is enabled by default in minikube.
+.. Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount]: `kubectl create serviceaccount tiller --namespace kube-system`.
+.. Bind it to the almighty cluster-admin role: `kubectl apply -f ./tiller-rbac.yaml`.
+.. Install tiller: `helm init --service-account tiller`.
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it use the `minikube addons enable ingress` command.
+* DNS discovery should be enabled. It is enabled by default in Minikube.
 
 [id="cluster-ip"]
-== Cluster IP
+== Obtaining the cluster IP
 
-* If your cluster is running on MiniKube, simply type `minikube ip` at your terminal
-* If your cluster is in the cloud, obtain the hostname or ip address from your cloud provider
+To obtain the cluster IP:
 
-In production, you should specify a hostname (see https://github.com/eclipse/che/issues/8694[here] why). In case you don’t have a hostname (e.g. during development), and would still want to use a host-based configuration, you can use services such as nip.io or xip.io.
+* If your cluster is running on MiniKube, type `minikube ip` at the terminal.
```

Comment:
"Terminal" is used here, but "command line" is used elsewhere in this documentation.